EiDiA_frontend/src/components/export/selectTemplate/TemplateList.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {List, Typography} from '@material-ui/core';
+import ExportService from '../../../services/ExportService';
+import TemplateListItem from './TemplateListItem';
+
+export default class TemplateList extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            templateList: [],
+            selectedIndex: 0 // index of marked/selected template
+        }
+        this.handleListItemClick = this.handleListItemClick.bind(this);
+    }
+
+    componentDidMount() {
+        let newState = this.state;
+        ExportService.getAllTemplates().then((data) => {
+            newState.templateList = data.exportTemplates;
+            this.setState(newState);
+        })
```

Comment:
es graucht auch ein catch
```suggestion
        })
        .catch(error => console.log(error));
```